lightning/src/ln/channelmanager.rs

```diff
@@ -1119,6 +1119,9 @@ pub(super) const CLTV_FAR_FAR_AWAY: u32 = 6 * 24 * 7; //TODO?
 // a payment was being routed, so we add an extra block to be safe.
 pub const MIN_FINAL_CLTV_EXPIRY: u32 = HTLC_FAIL_BACK_BUFFER + 3;
 
+/// Maximum total CTLV difference we allow for a full payment path
+pub const MAX_TOTAL_CLTV_EXPIRY_DELTA: u32 = 1000;
```

Comment:
This likely belongs in the `router` module, IMO. It's not specific to our channel logic or even defined in the spec, its just a limit on the router. Also re: naming: maybe call it "SUGGESTED" or "DEFAULT"?

Reply:
Alright, moved and renamed the limit with https://github.com/lightningdevkit/rust-lightning/pull/1234/commits/54a98eec3232ad4180ac17dc433316729d484159.